keps/prod-readiness/sig-instrumentation/1209.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1209
+beta:
+  approver: "@johnbelamaric"
```

Comment:
@johnbelamaric I couldn't get CI to pass for doing this refactor since this PR predates PRR... can you put a ceremonial approval on beta, which predates the process, and then the KEP authors can get a proper review when they work on stable graduation this release?

Reply:
/cc @wojtek-t 

in case @johnbelamaric can't get to it, (@wojtek-t would you mind helping us here)?